autoload/vebugger/jdb.vim

```diff
@@ -87,14 +87,16 @@ function! s:findFolderFromStackTrace(src,nameFromStackTrace,frameNumber)
   endif
 
   " If no such tag was found, try to find it using the src path.
-	let l:path=a:src
-	for l:dirname in split(a:nameFromStackTrace,'\.')
-		let l:nextPath=l:path.'/'.fnameescape(l:dirname)
-		if empty(glob(l:nextPath))
-			return l:path
-		endif
-		let l:path=l:nextPath
-	endfor
+  for l:one_path in vebugger#util#listify(a:src)
+    let l:path=l:one_path
```

Comment:
You could probably just use `l:path` as the iteration variable, but OK.

Reply:
Done.